scripts/run_commands

```diff
@@ -27,34 +28,42 @@ sha256sum miniconda.sh | grep $conda_chksum
 
 bash miniconda.sh -b -p /opt/conda
 rm -f miniconda.sh
+
+# Work around TLS issues while updating conda's python and retrieving channel notices
+export CONDA_NUMBER_CHANNEL_NOTICES="0"
 touch /opt/conda/conda-meta/pinned
 ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
 source /opt/conda/etc/profile.d/conda.sh
 conda activate
+conda config --set always_yes True
 conda config --set show_channel_urls True
 conda config --set channel_priority strict
 conda config ${additional_channel} --add channels conda-forge
 conda config --show-sources
-conda update --all --yes
-conda clean -tipy
 
-# Install conda build and deployment tools.
-conda install --yes --quiet \
-    git patch \
-    python=3.9 pip \
-    mamba boa \
-    conda-build anaconda-client
+# Update everything first
+mamba update --all
```

Comment:
Should we constrain `python` here too? Know we weren't before, but wondering if we should change that

```suggestion
mamba update --all \
    "python=$python_version"
```

Reply:
The `update` command encodes logic to leave `python` in its current `maj.min` version (only changes patch versions), so we don't need to (and would actually throw an error because `update` cannot take matchspecs with versions).